packages/ai/src/Button.ts

```diff
@@ -1,16 +1,24 @@
 import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import { renderFinished } from "@ui5/webcomponents-base/dist/Render.js";
 import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
 import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import event from "@ui5/webcomponents-base/dist/decorators/event.js";
+import slot from "@ui5/webcomponents-base/dist/decorators/slot.js";
 import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
-
-// Deps
 import MainButton from "@ui5/webcomponents/dist/Button.js";
+import Icon from "@ui5/webcomponents/dist/Icon.js";
+import ButtonDesign from "@ui5/webcomponents/dist/types/ButtonDesign.js";
+import ButtonState from "./ButtonState.js";
 
-// Template
-import template from "./generated/templates/ButtonTemplate.lit.js";
+import ButtonTemplate from "./generated/templates/ButtonTemplate.lit.js";
 
 // Styles
-import styles from "./generated/themes/Button.css.js";
+import ButtonCss from "./generated/themes/Button.css.js";
+
+// Animation durations constants
+const AI_BUTTON_FADE_OUT_DURATION = 180;
+const AI_BUTTON_FADE_IN_DURATION = 60;
+const AI_BUTTON_FADE_RESET_DURATION = 160;
```

Comment:
Do we need them global for the file? We can just have constants in the methods using camel case for naming. We don't need such varaibles globally

Reply:
Agree.